councilmatic_core/management/commands/import_data.py

```diff
@@ -459,10 +459,7 @@ def grab_bills(self):
 
         os.makedirs(self.bills_folder, exist_ok=True)
 
-        if hasattr(settings, 'OCD_CITY_COUNCIL_ID'):
-            query_params = {'from_organization__id': settings.OCD_CITY_COUNCIL_ID}
-        else:
-            query_params = {'from_organization__name': settings.OCD_CITY_COUNCIL_NAME}
+        organization_ids = [(o.ocd_id, o.name) for o in Organization.objects.all()]
 
```

Comment:
I like this approach. Would it be worthwhile to implement the same logic in [grab_organizations](https://github.com/datamade/django-councilmatic/blob/master/councilmatic_core/management/commands/import_data.py#L329)? Or to make organization_ids a reusable variable (somewhere at the top of the file) and use it here in `grab_organizations`?

Reply:
I think this mainly works because it happens sequentially after `grab_organizations`. If you don't have any organizations yet, I'm not sure it would really work there. Unless I'm missing something. 

The main thing I'm not super happy with is that, in most councilmatics, most of the organizations don't appear as the `from_organization` for any bills so this ends up doing a lot of work unnecessarily. But, I suppose it'll make sure to get everything should something suddenly appear.

Reply:
Yes, you are right! I think this makes sense, then.